### ISSUE TITLE
Lose control of CCA

### ISSUE THREAD

**Issue by menovak** [ThePacielloGroup/CCA-Win#24]
After I use CCA once by picking a foreground color, then when I hover mouse over the CCA application to select another color (foreground or background), my mouse cursor disappears when over the CCA application window, making it difficult to select any options.   

Running in Windows 8.1, 64 bit, version 2.4 (according to Help window) downloaded 9-12-2016.

**Comment by ghost** [comment]
I can't reproduce it yet. but, I did a small update.
Could you try the latest binary? https://github.com/ThePacielloGroup/CCA-Win/tree/develop/package

**Comment by menovak** [comment]
hi Jun,

If I go to that URL (github page), what is the best way to download the
file?

Mark

**Comment by stevefaulkner** [comment]
@menovak do you have a github client installed?

**Comment by menovak** [comment]
yes, have github, sorry wasn't thinking about that, will grab a copy and exercise it during the next audit, but will be a couple of weeks to test as next week is all meetings...

**Comment by menovak** [comment]
ok, i've tried this latest CCA, and it has the same issues I reported above, I can click and get one color (say foreground), but then when I try to click on the other (say background), I am unable to select it as the mouse cursor disappears when mousing over CCA.    I am successfully using an older version of CCA, think it is version 2.2a, according to the Help:About menu.

**Comment by ghost** [comment]
Thank you for your report.  I will modify some functions.

**Comment by ghost** [comment]
updated. could you try it again? https://github.com/ThePacielloGroup/CCA-Win/commit/b8445bedc4f310fb12535a06553efbc5ad1d5825

**Comment by menovak** [comment]
hi Jun

I've grabbed a copy of CCA and tested it again in both Windows 8.1 and
Windows 10, both 64 bit, using IE 11, and still see the same behavior.

When CCA has focus, I click on the foreground eye-dropper and then go
select an item in IE 11 to get the color, that is fine. I then select the
background eye-dropper, and it does not provide any select window, and from
that point, if CCA has focus, the mouse cursor disappears when within the
CCA window, meaning I can not see where the mouse pointer is to select any
options within the CCA window.

I also checked using FF 49 on Windows 8.1, similar problem.

I also checked Chrome 54 on Windows 10, and CCA worked one time ( I was
able to select both a foreground and background), but then the same problem
appeared.

Hope this description helps.   I am using an older version of CCA which
still works fine for my testing in the meantime, so there is no rush to
figure this out for me.

Regards,
Mark

**Comment by ghost** [comment]
The issue hasn't been resolved yet. I'm continuing the cause analysis.

**Comment by patrickhlauke** [comment]
@menovak any chance you could test the latest released version and let us know if it's still exhibiting the same funky problems?
